### DISCUSSION TITLE
How to Earn the Arctic Code Vault Contributor Badge?

### DISCUSSION THREAD

**Discussion by alidehkhodaei** [community/community#55305] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, Can I still get the Arctic Code Vault Contributor badge?

**Comment by Devanshu-17** [comment, marked as answer]
Hey, @alidehkhodaei 

The GitHub Arctic Code Vault is a part of the GitHub Archive Program, a collaboration between Microsoft’s GitHub and groups like the Internet Archive, which hosts repository snapshots in its Wayback Machine. This was done in honour of storing the users’ contributing in open-source community.

So the answer is **NO** because this badge was given to the people who were as a user in GitHub and had open-source contributed to any repo there before the data of all users were stored in Artic Vault before February 2, 2020. Unfortunately, even I was not a part of it😔.

**Comment by alidehkhodaei** [comment]
Thanks for your answer.

**Comment by SoumyadipYT-OSS** [comment]
GitHub Arctic Code Vault last awared in 2020, after that this badge is discontinued by GitHub community for keep it one of the legendary badge achievement. GitHub Arctic code is such if you contribute your knowledge or code in open source community with a great feedback and if your code is really helpful for the next generation then they award you with this badge for your github repo and your contribution.

**Comment by nguyentandat1302** [comment]
I want to contribute please
